app/helpers/easter_egg_helper.rb

```diff
@@ -0,0 +1,29 @@
+module EasterEggHelper
+  def easter_egg_class
+    class_names(
+      december: december?,
+      'may-4th': may_4th?
+    )
+  end
+
+  def easter_egg_logo
+    case
+    when december?
+      'december/logo_full_small.png'
+    when may_4th?
+      'may_4th/logo_full_small.png'
+    else
+      false
```

Comment:
```suggestion
      nil
```

Functionally the same but `nil` is more consistent since this method is expected to return an object instead of boolean